udev/69-hid.rules

```diff
@@ -16,17 +16,20 @@ SUBSYSTEM=="usb", ATTRS{idVendor}=="04d8", ATTRS{idProduct}=="003f", TAG+="uacce
 KERNEL=="hidraw*", ATTRS{idVendor}=="04d8", ATTRS{idProduct}=="003f", TAG+="uaccess"
 
 # Once done, optionally rename this file for your application, and drop it into
-# /etc/udev/rules.d. Note that these rules must have priorty before 70-uaccess.rules
-# for example, name the file /etc/udev/rules.d/69-my-application-hid.rules.
+# /etc/udev/rules.d/.
+# NOTE: these rules must have priorty before 73-seat-late.rules.
+# (Small discussion/explanation in systemd repo:
+#  https://github.com/systemd/systemd/issues/4288#issuecomment-348166161)
+# for example, name the file /etc/udev/rules.d/70-my-application-hid.rules.
```

Comment:
```suggestion
# for example, name the file /etc/udev/rules.d/72-my-application-hid.rules.
```

Reply:
> How about renaming the file to 72-hid.rules?

So there is a standard set of rules that adds a `uaccess` TAG (`70-uaccess.rules`) with priority `70`.
I'm thinking maybe having and example with the same priority would be nit.